kafka/modules/kafka/2.3.0/install.sh

```diff
@@ -6,9 +6,10 @@ SOURCES_DIR=/tmp/artifacts
 LICENSE_DIR=/root/licenses
 PRODUCT_LICENSE_DIR=${LICENSE_DIR}/${COM_REDHAT_COMPONENT}
 
-tar xvfz "${SOURCES_DIR}/kafka.tar.gz" -C ${KAFKA_HOME} --strip-components=1
-
-cp -r ${SOURCES_DIR}/opentracing-kafka-client.jar ${KAFKA_HOME}/libs/
+# Copy contents of zip without the root dir
+TMP=$(zipinfo -1  ${SOURCES_DIR}/kafka.zip | grep -oE '^[^/]+' | uniq)
```

Comment:
we are really missing the strip-components option that tar tool has :-(

Reply:
Yea, unfortunately unzip doesn't have the "--strip-components=1" equivalent. This is the best we can do with unzip from what I understand. @scholzj we are tied to zips for the Kafka dependencies, correct?